src/pages/workspace/WorkspaceSettingsPage.js

```diff
@@ -55,20 +56,31 @@ class WorkspaceSettingsPage extends React.Component {
         if (this.props.policy.isPolicyUpdating) {
             return;
         }
-        const name = values.name.trim();
         const outputCurrency = values.currency;
-        Policy.updateGeneralSettings(this.props.policy.id, name, outputCurrency);
+        Policy.updateGeneralSettings(this.props.policy.id, values.name, outputCurrency);
         Keyboard.dismiss();
     }
 
     validate(values) {
         const errors = {};
-        if (!values.name || !values.name.trim().length) {
+        const name = values.name.trim();
+
+        if (this.hasHtml(name)) {
+            errors.name = 'HTML tags are not allowed in workspace names.';
```

Comment:
I think we we should use `props.translate` here? I'd ask for the Spanish version in https://expensify.slack.com/archives/C21FRDWCV

Reply:
Oh Whoops